akka-actor-typed/src/main/scala/akka/actor/typed/TypedActorContext.scala

```diff
@@ -15,7 +15,7 @@ import akka.annotation.ApiMayChange
  */
 @DoNotInherit
 @ApiMayChange
-trait ActorContext[T] {
+trait TypedActorContext[T] {
```

Comment:
I don't have better suggestions so I'm fine with this name. In general we should be careful to not sprinkle `Typed` in all names because we already have the package name `typed` and it becomes just to much Typed everywhere. However, for this class it can be fine because it shouldn't be much visible to end users.

Reply:
I think this could be easily fixed by import as `TypedActorContext `, as what @patriknw there are already a `typed` in the package name.

Reply:
And there is an old friend `TypedActor` out there too.

Reply:
This is to solve a problem with naming conflicts in Java code, which doesn't support importing into a different name or relative package qualifiers.

Reply:
Exactly!

Reply:
Ah... Java... yes.

Reply:
@hepin1989 they can't be solved via imports if you mean { Actual => Usable } because it is too much boilerplate - 139 uses of the class in the codebase.

Reply:
> @hepin1989 they can't be solved via imports if you mean { Actual => Usable } because it is too much boilerplate - 139 uses of the class in the codebase. And what they said, Java :)

Yes, Java :) I have to say I was wondering too... there are three `ActorContext[T]`s.